tests/browser/test/test_setup.js

```diff
@@ -73,6 +73,11 @@ const testFileLocations = {
   playground: 2,
 };
 
+const screenDirection = {
+  rtl: -1,
+  ltr: 1,
+};
```

Comment:
This could have a JSDoc marking it as an `@enum` (as well as describing it, if useful).

Please use `CONSTANT_CASE` for the enum values (`LTR`, `RTL`).